### PR TITLE
Add deployment code into monorepo

### PR DESCRIPTION
**Pull request by rob-3**
- feat: add some necessary changes for deployment
- feat: add deployment into monorepo

### CODE REVIEW

**Comment by rob-3** [comment]
This is ready for review, I just need to update the deployment server manually one last time and update the secrets on GitHub so the build passes.

I'm mainly looking for feedback on how to handle the `.prettierrc` and related config files, and whether I've dropped the project into the right place. Feel free to also review the bulk of the deployment code at `repos/deployment/src/routes/api/github/webhook/+server.ts`, however.

File: repos/deployment/src/routes/api/github/webhook/+server.ts [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+						spawn('kill $(lsof -ti :5000) && node', [deploymentBuildPath], {
+							detached: true,
+							shell: true,
+							stdio: 'ignore',
+							env: {
+								HOST: '127.0.0.1',
+								PORT: '5000',
+							}
+						});
+						// we should be killed momentarily
```
[the remark this answers is not in the document]
Comment (rikimbili) [reply repos/deployment/src/routes/api/github/webhook/+server.ts:365]: I would just use their cli, not build on top of their code. They have a paid version as well, so I assume they might not be too happy about you using their code lol

[the remark this answers is not in the document]
Comment (rikimbili) [reply repos/deployment/src/routes/api/github/webhook/+server.ts:365]: But yea, if this works then let's not add more work and leave it as is.

File: repos/deployment/.eslintrc.cjs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (rob-3) [reply repos/deployment/.eslintrc.cjs:1]: Oh - I was wondering about this. I notice you do `file:*` instead of `workspace:*`. Any reason?

**Review by rikimbili — APPROVED** [review]

**Pull request merged**